.github/workflows/main.yml

```diff
@@ -108,12 +108,12 @@ jobs:
         with:
           ref: main
           fetch-depth: 0
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
         with:
           node-version: 12.x
       - name: Create auto-tag
         if: ${{ success() }}
         run: npx @eqworks/release tag --github -v
         env:
           GITHUB_OWNER: EQWorks
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.CD_GITHUB_TOKEN }}
```

Comment:
supposedly, the repo auto-provisioned `${{ secrets.GITHUB_TOKEN }}` is better since the `CD_` one is provisioned by humans. any other reason for the change (similarly for all other places where it has been changed to the `CD_` version)? @vxsl

Reply:
oh, I see, I thought it was the other way around (`CD_` is auto-provisioned because it's global). Will drop this commit